controllers/intergration_test.go

```diff
@@ -91,9 +88,6 @@ func setupControllersAPIServerAndClient(t *testing.T, parentCtx context.Context,
 		Logger:             logger,
 		LeaderElection:     false,
 		MetricsBindAddress: "0",
-		Controller: ctrlconfigapi.ControllerConfigurationSpec{
```

Comment:
In the latest controller-runtime version, this CacheSyncTimeout field is not optional anymore.

Reply:
Do you mean that CacheSyncTimeout is now optional?

Reply:
No, it has always been optional, but we provided a value to make sure we could use the value elsewhere.
I now replaced this logic with the defaulting logic here: https://github.com/cert-manager/issuer-lib/pull/14/files#diff-be55a9848e35ac4afc302b867688edc0fea66401d35f31e62cb8f32eb5e4372bR443-R444
This makes it a lot easier to reason about.